packages/web-components/src/drawer/drawer.ts

```diff
@@ -0,0 +1,244 @@
+import { attr, FASTElement, observable, Updates } from '@microsoft/fast-element';
+import { eventAnimationEnd, keyEscape } from '@microsoft/fast-web-utilities';
+import { DrawerModalType, DrawerPosition, DrawerSize, DrawerType } from './drawer.options.js';
+
+export class Drawer extends FASTElement {
+  private closing: boolean = false;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+
+    Updates.enqueue(() => {
```

Comment:
Perhaps add a comment on why this is necessary given that open is initialized to false and openChanged already calls show(). Is it because the initial openChanged may occur before the dialog ref is set?

Reply:
I've added a comment explaining the reasoning here